### PR TITLE
Tokyo CFP

### PR DESCRIPTION
**Pull request by spesnova**
## What does this PR do?
Prepare the CFP page in Tokyo 2020 site.

### How to see the site on my laptop?

```console
$ git clone git@github.com:k8sjp/kubernetes-community-days.git && cd kubernetes-community-days
$ git checkout spesnova/tokyo-cfp
$ yarn
$ hugo server --buildDrafts --buildFuture
$ open http://localhost:1313/events/2020-tokyo/cfp
```

## Motivation
We're going to start the CFP process soon. Then we need the CFP page and routing for it.

## To-dos

- [x] Update `cfp_deadline` in `_index.md`  
- [x] Add description to route people to the CFP page in `_index.md` (Overview page)
- [x] Update description in `cfp.md`  (e.g. How to submit?)

### CODE REVIEW

**Comment by spesnova** [comment]
@Ladicle May I ask you to take over this PR since you are assigned to procced CFP part.

**Comment by Ladicle** [comment]
Sure, I'll take it.

File: content/events/2020-tokyo/cfp.md [hunk 1]
```diff
@@ -41,10 +39,10 @@ Note: All submissions with 3–5 speakers are required to have at least one spea
...
 * Provide resources to enhance your proposal. These can be videos of you or your speakers presenting elsewhere, links to personal websites (including LinkedIn), links to your open source projects, or published books.
 * If you choose to submit a tutorial please explicitly mention what the audience will learn from or walk away with after attending your session. Additionally, please indicate what prerequisites (if any) are needed for the attendee to know prior to attending, and if any materials should be brought with them or downloaded ahead of time (i.e. must install Python 2.7.15) prior to attending.
-* Submit speaking session to <name>, <email address>. 
-
-# Code of conduct
+* Submit speaking session to <name>, <email address>.
 
-The Kubernetes Community Days are dedicated to providing a  harassment-free experience for participants at all of our events. We encourage all submitters to review our complete [Code of Conduct](/code-of-conduct).
+# 行動規範 (Code of conduct) について
+Kubernetes Community Daysは、すべての参加者にハラスメントフリーな体験を提供したいと思っています。CFP をご提出いただく際には [Code of Conduct](https://kubernetescommunitydays.org/code-of-conduct/) を確認し、行動規範に沿ってコンテンツを作成してください。(現在日本語版 Code of Conduct もです)。
```
Comment (superbrothers) [inline content/events/2020-tokyo/cfp.md]:
```suggestion
Kubernetes Community Daysは、すべての参加者にハラスメントフリーな体験を提供したいと思っています。参加者の方は [行動規範（Code of Conduct）](https://www.linuxfoundation.jp/code-of-conduct/) を確認してください。
```

**Comment by Ladicle** [comment]
## _index.md
image

## cfp.md
image

**Review by superbrothers — APPROVED** [review]

**Pull request merged**